README.md

```diff
@@ -2,6 +2,7 @@
 
 [![CircleCI](https://circleci.com/gh/Seneca-CDOT/telescope.svg?style=svg)](https://circleci.com/gh/Seneca-CDOT/telescope)
 [![js-airbnb/prettier-style](https://img.shields.io/badge/code%20style-airbnb%2Fprettier-blue)](https://github.com/airbnb/javascript)
+![Node.js CI](https://github.com/Seneca-CDOT/telescope/workflows/Node.js%20CI/badge.svg)
```

Comment:
The badge status says failing for me, is this something wrong with the branch ?

![image](https://user-images.githubusercontent.com/35276477/98844432-4e84ae80-241a-11eb-9e89-2eccfae7b01e.png)

Reply:
I don't have the workflow set up for my fork/branch, but I wouldn't think it would matter since I'm referencing the main branch of Telescope. Looking into this further now

Reply:
Could it be because we aren't explicitly referencing the master branch ?
checkout the `branch` parameter
https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter

Reply:
Even when I go to create the badge straight from the workflow, it reflects that it is failing...

![image](https://user-images.githubusercontent.com/37879416/98845629-fbabf680-241b-11eb-86db-dd802529461c.png)

Reply:
> Could it be because we aren't explicitly referencing the master branch ?
> checkout the `branch` parameter
> https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter

Referencing the master branch gives "No status":

![image](https://user-images.githubusercontent.com/37879416/98845772-301fb280-241c-11eb-8ece-49b4c54462a5.png)

Reply:
I have also tested out various events, such as the <code>pull_request</code> event on the master branch, but had similar results:

![image](https://user-images.githubusercontent.com/37879416/98849110-b2aa7100-2420-11eb-865d-b4162ea11ca7.png)